float32 x_min
float32 y_min
float32 width
float32 height
float32 confidence